src/Aspire.Hosting/Dashboard/DashboardLifecycleHook.cs

```diff
@@ -17,8 +20,14 @@ internal sealed class DashboardLifecycleHook(IConfiguration configuration,
                                              IOptions<DashboardOptions> dashboardOptions,
                                              ILogger<DistributedApplication> distributedApplicationLogger,
                                              IDashboardEndpointProvider dashboardEndpointProvider,
-                                             DistributedApplicationExecutionContext executionContext) : IDistributedApplicationLifecycleHook
+                                             DistributedApplicationExecutionContext executionContext,
+                                             ResourceNotificationService resourceNotificationService,
+                                             ResourceLoggerService resourceLoggerService,
+                                             ILoggerFactory loggerFactory) : IDistributedApplicationLifecycleHook, IAsyncDisposable
 {
+    private readonly CancellationTokenSource _shutdownTcs = new();
```

Comment:
```suggestion
    private readonly CancellationTokenSource _shutdownCts = new();
```
And everywhere else.

Reply:
oops